censors complionator sound

KoboldCommando opened the pull request:
## About The Pull Request

I missed one.  As it turns out someone at tg copied, renamed, and hid insult.ogg just so the compli-o-nator can continue to shout swears.  What the actual fuck.

## Why It's Good For The Game

consistency and rule breaks

## Changelog

:cl:
fix: removed the sec hailer's ability to break server rules, for real this time
/:cl:

Iamgoofball commented:
you forgot to write Why It's Good For The Game

dwasint commented:
> you forgot to write Why It's Good For The Game

![image](https://github.com/Monkestation/Monkestation2.0/assets/82520990/841b7273-38a3-439a-ab18-dca2b3e75f3e)

KoboldCommando commented:
> you forgot to write Why It's Good For The Game

Thank you for the reminder!  But it looks like I did in fact remember to write one.

I can elaborate if it helps, 
- consistency: similar rule-breaking features were recently fixed, but this one slipped through the cracks.  This PR resolves that inconsistency. 
- rule breaks: we have a policy against overly hateful or sexual speech, and would likely punish someone who repeated the beepsky rant, so to avoid hypocrisy it makes sense to edit the rant itself.

Hope this clears things up!

DexeeXI commented:
still rest in peace Beepsky.

Iamgoofball commented:
@KoboldCommando I just did a quick flip through the rules, what specifically is objectionable in the beepsky rant?

Additionally, your About The Pull Request was incorrect, we moved the complionator audio files during routine maintenance to fix docker builds in https://github.com/tgstation/tgstation/pull/51781 per git history.

DexeeXI commented:
@Iamgoofball https://www.youtube.com/watch?v=Hu3D1yH2WYM reference material.

Iamgoofball commented:
i am familiar with the words of beepsky's rant, i've been here for nearing 12 years

KoboldCommando commented:
> @KoboldCommando I just did a quick flip through the rules, what specifically is objectionable in the beepsky rant? It's mostly just vulgar word salad on par with a 12 year old repeating every swear they can think of, if there was any specific slurs in there I'd get it but it's frankly just vulgar screaming.
> 
> Additionally, your About The Pull Request was incorrect, we moved the complionator audio files during routine maintenance to fix docker builds in [tgstation#51781](https://github.com/tgstation/tgstation/pull/51781) per git history.

When I fixed insult.ogg on Beecode the compli-o-nator pointed to the same sound file as beepsky.  I didn't pick up on the fact that it had been duplicated, renamed, and slightly obscured in the code.  My "about" is not exactly incorrect, but the presumption of intent is indeed facetious and not to be taken seriously.

As for the rule break specifically, I feel like it should be obvious, but the easiest rule to point to is 3. No Sexual Content. specifically "references to genitalia and sexual acts".  It's additionally extremely graphic, and while we don't have a specific rule against that it's still a line that can be crossed.
